canonical/lxd-ui
# chore: add tests for permissions

## Done

- Added tests for permissions feature

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @mas-who or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - Ensure CI passes
    - Run permissions tests locally and make sure it passes.
